spec/lib/efile/nj/nj1040_calculator_spec.rb

```diff
@@ -1045,6 +1045,292 @@ def over_65_birth_year
     end
   end
 
+  describe 'line 59 - excess UI/WF/SWF' do
+    context 'with excess contribution but only one w2' do
+      let(:intake) { create(:state_file_nj_intake, :df_data_box_14) }
+      it 'does not fill line 59' do
+        w2 = intake.state_file_w2s.first 
+        w2.update_attribute(:box14_ui_wf_swf, 0)
+        w2.update_attribute(:box14_ui_hc_wd, described_class::EXCESS_UI_WF_SWF_UI_HC_WD_MAX + 1)
+        instance.calculate
+        expect(instance.lines[:NJ1040_LINE_59].value).to eq(nil)
+      end
+    end
+    
+    context 'with multiple w2s but excess contribution only from one employer' do
+      let(:intake) { create(:state_file_nj_intake, :df_data_box_14) }
+      before do
+        create :state_file_w2, state_file_intake: intake 
+      end
+
+      it 'does not fill line 59' do
+        expect(intake.state_file_w2s.count).to eq(2)
+        first_w2 = intake.state_file_w2s.first 
+        second_w2 = intake.state_file_w2s.all[1]
+        first_w2.update_attribute(:box14_ui_wf_swf, 0)
+        first_w2.update_attribute(:box14_ui_hc_wd, described_class::EXCESS_UI_WF_SWF_UI_HC_WD_MAX + 1)
+
+        expect(first_w2.box14_ui_wf_swf).to eq(0)
+        expect(first_w2.box14_ui_hc_wd).to eq(described_class::EXCESS_UI_WF_SWF_UI_HC_WD_MAX + 1)
+        expect(second_w2.box14_ui_wf_swf).to eq(nil)
+        expect(second_w2.box14_ui_hc_wd).to eq(nil)
```

Comment:
[pebble] ditto the above comment: this seems like it's just checking that setting the attribute 2 lines earlier worked